docker/syntax_check.sh

```diff
@@ -6,7 +6,7 @@ CLANG_FORMAT_BIN="/usr/bin/clang-format-12"
 
 PROJECT_DIR=$(pwd)
 
-cd $PROJECT_DIR
+cd "$PROJECT_DIR" || exit 
```

Comment:
why?

Reply:
Prevent next commands run in script after incorrect change directory.

Reply:
why not `exit 255` in that case?